Content.Tests/DMProject/Tests/Builtins/JsonEncode.dm

```diff
@@ -13,3 +13,5 @@
 	ASSERT(json_encode(list("A" = 3, "B" = 5)) == @'{"A":3,"B":5}')
 	ASSERT(json_encode(matrix(1,2,3,4,5,6)) == @'[1,2,3,4,5,6]')
 	ASSERT(json_encode(/datum/proc/foo) == "\"/datum/proc/foo\"")
+	ASSERT(length(json_encode(list("name"="fridge", "power"=12))) == 28)
```

Comment:
crlf vs lf?

Reply:
This is the issue. `Utf8JsonWriter` uses `Environment.NewLine` which is `\n` on Linux and `\r\n` on Windows. Unfortunately I don't see any way to change this behavior, so we either have to do an inefficient text replace or decide not to care about the final length of `json_encode()`.